src/Components/ChartCard/ChartCard.js

```diff
@@ -46,8 +51,13 @@ const ChartCard = ({ sysStats, chartStats }) => {
                             domainPadding={{ x: [30, 25] }}
                             height={300}
                             width={550}
-                            minDomain={{ y: 0 }}
                         >
+                            <ChartAxis
+                                dependentAxis
+                                domain={[0, maxYAxisValue(chartStatsData.timeSeriesStatsList)]}
```

Comment:
A minor nitpick, we need to make sure that we always receive `chartStatsData` as defined and its type is Object. Otherwise, it may lead to type/reference errors

Reply:
@mkholjuraev so something like `domain={[0, maxYAxisValue(chartStatsData?.timeSeriesStatsList || 4)]}` ?

Reply:
Yes, this also works.